test/Dapr.AspNetCore.IntegrationTest.App/CustomTopicAttribute.cs

```diff
@@ -30,5 +30,7 @@ public CustomTopicAttribute(string pubsubName, string name)
         public new string Match { get; }
 
         public int Priority { get; }
+
+        public string DeadLetterTopic { get; }
```

Comment:
This change isn't required unless you want to use it in your test.

Reply:
I had to add it because I had added `DeadLetterTopic` to `ITopicMetadata`.